Objects/ComingObjects/ComingObject.gd

```diff
@@ -5,14 +5,16 @@ signal timeout
 func _ready():
 	connect("timeout",self,"_on_timeout")
 	
-	var elf_pos = get_node("/root/World").find_node("Elf").global_position
-	var diff = global_position.x - elf_pos.x
+	var cam_pos = get_node("/root/World").find_node("Camera2D").position
```

Comment:
necessary

Reply:
Hmm, because entering tavern doesn't happen when elf is exactly on tavern's doors?